### DISCUSSION TITLE
Progress bar / Completion

### DISCUSSION THREAD

**Discussion by RezaODB** [livewire/flux#847] [Feature Requests] [answered]
Would be nice to have! Similar to what we have in tailwind components: https://tailwindui.com/components/application-ui/navigation/progress-bars

**Comment by Hesesses** [comment]
These would be great 👍

**Comment by Frankie813** [comment]
I would love to see a progress bar component added. I find a lot of use cases for progress bars and this would be great +1

**Comment by bobik1337** [comment]
Agree on this. Would find a use for it

**Comment by sammyaxe** [comment]
Progress bar component would be immensely useful, especially now that we have slider component - sometimes you want to display loading state in terms of % etc

**Comment by calebporzio** [comment, marked as answer]
We have progress bars now: https://fluxui.dev/components/progress

And we also have a timeline component: https://fluxui.dev/components/timeline
